lib/pure/typetraits.nim

```diff
@@ -64,8 +64,17 @@ proc supportsCopyMem*(t: typedesc): bool {.magic: "TypeTrait".}
   ##
   ## Other languages name a type like these `blob`:idx:.
 
-proc isNamedTuple*(T: typedesc): bool {.magic: "TypeTrait".}
-  ## Return true for named tuples, false for any other type.
+proc isNamedTuple(T: typedesc): bool =
```

Comment:
```suggestion
proc isNamedTuple*(T: typedesc): bool =
```
This should be exported right?

Reply:
Oops, thanks for checking

Reply:
np, I think it may be the cause of the CI failures..